collect_app/src/androidTest/java/org/odk/collect/android/support/rules/BlankFormTestRule.kt

```diff
@@ -64,6 +66,6 @@ class BlankFormTestRule @JvmOverloads constructor(
             val projectId = DaggerUtils.getComponent(application).currentProjectProvider()
                 .getCurrentProject().uuid
 
-            return FormNavigator.newInstanceIntent(application, projectId, form!!.dbId)
+            return FormNavigator.newInstanceIntent(application, FormsContract.getUri(projectId, form!!.dbId), FormFillingActivity::class)
```

Comment:
But then we end up with a stack of activities `FormUriActivity` > `FormFillingActivity` and it's a problem with the code you have recently added for destroying activities in tests. `FormUriActivity` would be destroyed instead of `FormFillingActivity` or it would even throw and exception before:

```
   fun destroyActivity(): FormEntryActivityTestRule {
        lateinit var scenarioActivity: Activity
        scenario.onActivity {
            scenarioActivity = it
        }

        if (ActivityHelpers.getActivity() != scenarioActivity) {
            throw IllegalStateException("Can't destroy backstack!")
        }

        scenario.moveToState(Lifecycle.State.DESTROYED)
        return this
    }
```